app/design/frontend/Magento/blank/requirejs-config.js

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
That's a good point and I agree with you. I've updated the code and tested it.

Thanks :)